projects/core/components/root/root.style.less

```diff
@@ -13,15 +13,17 @@
     }
 }
 
-html,
-.tui-zero-scrollbar {
-    .scrollbar-hidden();
-}
+html[data-tui-theme] {
+    &,
+    .tui-zero-scrollbar {
```

Comment:
`.tui-zero-scrollbar` should probably work regardless of the attribute.

Reply:
fixed